include/cuco/static_map.cuh

```diff
@@ -384,9 +392,9 @@ class static_map {
    *
    * Writes a `bool` to `(output + i)` indicating if the key `*(first + i)` exists in the map.
    *
-   * Hash should be callable with both `std::iterator_traits<InputIt>::value_type` and Key type.
-   * `std::invoke_result<KeyEqual, std::iterator_traits<InputIt>::value_type, Key>` must be
-   * well-formed.
+   * Hash should be callable with both <tt>std::iterator_traits<InputIt>::value_type</tt> and Key
+   * type. <tt>std::invoke_result<KeyEqual, std::iterator_traits<InputIt>::value_type, Key></tt>
+   * must be well-formed.
```

Comment:
What's the deal with `<tt>`? afaik, markdown backticks work in Doxygen.

Reply:
Here's an example warning message of using backticks https://gpuci.gpuopenanalytics.com/blue/organizations/jenkins/nvidia%2Fgpuci%2FcuCollections%2Fprb%2FcuCollections-style/detail/cuCollections-style/147/pipeline#log-176. This happened exclusively to all docs containing the keyword "value_type".

Reply:
Which error in particular should I be looking at?

Reply:
Here's what I found:
```cpp
* `std::iterator_traits<InputIt>`
```
Works fine but adding `::value_type` will fail with `explicit link request to 'value_type' could not be resolved`:
```cpp
* `std::iterator_traits<InputIt>::value_type`
```
I thought the problem is that we didn't provide a link to the implementation of "std::iterator_traits" thus the link request to the code fails. So maybe marking a locally defined function with backticks will pass? Following this idea, I tried the below code but had no luck either:
```cpp
* `cuco::detail::Murmurhash3_32<key_type>::result_type`
```

Another thing I tried is turning off `AUTOLINK_SUPPORT` in the doxygen config file. We can use backticks without getting any warnings by doing that but I think our intention is to keep the auto-link on in Doxygen. Any thoughts?

Reply:
I suspect it's expecting docs to exist for `std::iterator_traits<InputIt>::value_type` to exist and link to or something.